core/src/main/groovy/noe/workspace/WorkspaceHttpdBaseOS.groovy

```diff
@@ -0,0 +1,47 @@
+package noe.workspace
+
+import groovy.util.logging.Slf4j
+import noe.common.utils.Library
+import noe.rhel.server.httpd.HttpdBaseOS
+
+@Slf4j
+class WorkspaceHttpdBaseOS extends WorkspaceAbstract {
+
+  WorkspaceHttpdBaseOS() {
+
+    if (platform.isRHEL8() || platform.isRHEL7()) {
+      HttpdBaseOS server = new HttpdBaseOS(basedir, '2.4')
+      serverController.addServer("httpd-24-1", server)
+    } else if (platform.isRHEL6()) {
+      HttpdBaseOS server = new HttpdBaseOS(basedir, '2.2')
+      serverController.addServer('httpd-22-1', server)
+    }
+  }
+
+  @Override
+  def prepare() {
+    log.debug("Creating Httpd BaseOS workspace.")
+    (serverController.httpdServerIds).each { String httpdId ->
+      /**
+       * Settings of mpm module, set MPM_MODULE variable to prefork(default), worker or event
+       */
+      def mpmModule = Library.getUniversalProperty('mpm.module', null)
```

Comment:
Not sure, but isn't `null` default anyway?

Reply:
It is more readable and strait forward with `null` definition in there. At least for me :)

Reply:
If you have no technical reason for that, please remove that, it is superfluous then.

Reply:
done